### DISCUSSION TITLE
Support Filter processor in the collector

### DISCUSSION THREAD

**Discussion by devshashankb** [open-telemetry/opentelemetry-demo#1540] [Q&A] [answered]
How can we add filter processor to collector to reduce the trace volume? We tried to just have it in collector, but it's not working. if any one tried it, can you please provide an example. Also it would be great to have more sampling configurations(Head and tail) added to the demo project.

**Comment by puckpuck** [comment, marked as answer]
A sampler would be used to reduce trace volume. The collector shipped with the demo does support the tail based sampler processor, and you can add the definition for it using the [otelcol-config-extras.yml](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/otelcollector/otelcol-config-extras.yml) file
